src/main.cpp

```diff
@@ -199,7 +201,9 @@ void mapMonitor(CMonitor* monitor)
             workspace = g_pCompositor->createNewWorkspace(i, monitor->ID);
```

Comment:
I would suggest not creating the workspaces, if persistent workspaces are off, because most of the workspaces will be deleted the next frame anyway, if they are not visible or have any windows on them.

Reply:
Good point, fixed.